src/server/endpoints/logoutHandler.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const HTTP_STATUSES = require('../modules/httpStatuses');
+const tokensCollection = require('../collections/tokens.js');
+const deleteAccessToken = tokensCollection.deleteDocumentByToken;
+
+function logout(req, res) {
+  res.setHeader('Content-Type', 'application/json');
+  let status;
+
+  if (req.header === null) {
+    let obj = {'errorType': 'HeaderMissing'};
+
+    status = HTTP_STATUSES.UNAUTHORIZED;
+    res.status(status).json(obj);
+  } else if (req.header.token === null) {
```

Comment:
- we should use the get/headers method according to the express [documentartion](http://expressjs.com/en/api.html#req.get).
- the HTTP standard uses the [Authorization](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization) header field to transfer the tokens. Of course we can use custom ones. Discuss it with your team.
- also according to the documentation the Authorization header field should/must contain a type before the token value.